lib/JsonLdDocumentLoader.js

```diff
@@ -18,8 +21,32 @@ class JsonLdDocumentLoader {
     this.documents.set(url, document);
   }
 
+  /**
+   * Adds a DID method specific driver to the cached resolver.
```

Comment:
What is a "cached resolver"?

Reply:
That's this thing: https://github.com/digitalbazaar/did-io#using-the-cachedresolver-to-get-did-documents-and-keys